extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -65,6 +71,17 @@ export class Renderer {
     });
   }
 
+  addLoadingBar() {
```

Comment:
I just noticed that we have not added this value in validator :( Please add here and ping validator folk to fast-track once submitted.

Reply:
done